data-prepper-plugins/otel-trace-source/src/test/java/com/amazon/dataprepper/plugins/source/oteltrace/OTelTraceSourceTest.java

```diff
@@ -54,6 +81,10 @@ private static String getUri() {
 
     @BeforeEach
     public void beforeEach() {
+        lenient().when(serverBuilder.service(any(GrpcService.class))).thenReturn(serverBuilder);
+        lenient().when(serverBuilder.http(anyInt())).thenReturn(serverBuilder);
+        lenient().when(serverBuilder.build()).thenReturn(server);
+        lenient().when(server.start()).thenReturn(completableFuture);
```

Comment:
TIL about `lenient()` - are some of these mocks not actually used somewhere (or is this necessary)?

Reply:
Yeah. Looks like strict stubbing check is new in Junit 5. For existing test cases these setup is not necessary, putting them in BeforeEach is to reduce lines of code in new test cases.